.github/workflows/new_release_branch.yml

```diff
@@ -41,4 +41,9 @@ jobs:
       - name: Cut release branch
         env:
           GITHUB_TOKEN:  ${{ secrets.GH_ACTIONS_PAT }}
-        run: ./release_branch.sh
+        run: |
+          if [[ "${{ needs.get-workflow-environment.outputs.environment_name }}" == "manual-release" ]]; then
```

Comment:
@timwright12 Can you double check this output var since you worked on it originally? I see that there's mixed usage between `environment_name` and `env_name` above where it's set around line 19.

Reply:
yeah I think it's OK, just getting remapped for this job